argocd/datadog_checks/argocd/check.py

```diff
@@ -0,0 +1,104 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from collections import defaultdict
+
+from six import PY2
```

Comment:
Do we need this if we drop python2 support?

Reply:
I think this is what we normally use to determine if the python running this check is 2 or not. If it's 2 then the check will raise the error here:

https://github.com/DataDog/integrations-core/blob/4ebc10adfb7ddd5e22f1ee199c18c89fc28f5bef/argocd/datadog_checks/argocd/check.py#L25-L29

So agent 6 that we're still releasing will have the option to change between python 2 and python 3. So this is mainly here to prevent users that use newest agent 6 to run this check.

Reply:
yea, sorry friday syndrome 🤦